framework/utils/utils.py

```diff
@@ -222,23 +222,44 @@ def convertMultipleToBytes(sizeString):
     except:
       raise IOError(UreturnPrintTag('UTILITIES')+': ' +UreturnPrintPostTag('ERROR') + '->  can not understand how to convert expression '+str(sizeString)+' to number of bytes. Accepted Mb,Gb,Kb (no case sentive)!')
 
-def stringsThatMeanTrue():
+# I don't think there's a reason to make this an enum, but it could be done.
+trueThingsFull = ('True', 'Yes', '1')
+trueThings = tuple(x[0].lower() for x in trueThingsFull)
+trueThingLegacy = ['yes','y','true','t','si','vero','dajie','oui','ja','yao','verum', 'evet', 'dogru', '1', 'on']
```

Comment:
I feel left out :)
yes in Arabic is: na'am
and no is: laa

Reply:
Hm, @alfoa is the Keeper of All Things That Mean True, so we can add more with his signoff. I included the list that we previously checked against, that an astute reviewer will ask me to remove since it isn't used anywhere :D

Reply:
I was just kidding. None of you guys will be able to pronounce it anyways. :)